grid-proxy/internal/explorer/db/postgres.go

```diff
@@ -31,8 +31,107 @@ var (
 	ErrContractNotFound = errors.New("contract not found")
 )
 
+var views = []string{
```

Comment:
This can instead by done with https://pkg.go.dev/embed package
this way the view can live in a separate file (say view.sql) and is embedded in the code so u can use it